pitop/battery/battery.py

```diff
@@ -63,26 +63,26 @@ def get_full_state(cls):
         return response.parameters()
 
     @property
-    def charging(self):
+    def is_charging(self):
```

Comment:
This might break things elsewhere so we should be careful to review. Examples, sys oled, etc.